bundles/org.openhab.automation.jrubyscripting/src/main/feature/feature.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.automation.jrubyscripting-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+	<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+
+	<feature name="openhab-automation-jrubyscripting" description="JRuby Scripting" version="${project.version}">
+		<feature>openhab-runtime-base</feature>
+		<bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.automation.jrubyscripting/${project.version}</bundle>
+		<configfile finalname="${openhab.conf}/services/jruby.cfg" override="false">mvn:org.openhab.addons.features.karaf/org.openhab.addons.features.karaf.openhab-addons-external/${project.version}/cfg/jruby</configfile>
```

Comment:
I think this configfile can now be removed, because it has become optional.

```suggestion
```

Reply:
Removing that doesn't prohibit configuration via a file, correct?

Reply:
You can configure `ConfigurebleService`s using both the UI and files.
File based configuration will overwrite configurations made in the UI.

Reply:
> I think this configfile can now be removed, because it has become optional.
> `<configfile finalname="${openhab.conf}/services/jruby.cfg" override="false">mvn:org.openhab.addons.features.karaf/org.openhab.addons.features.karaf.openhab-addons-external/${project.version}/cfg/jruby</configfile>`


@wborn without this line, the configuration file will require prefixing every line with `org.openhab.automation.jrubyscripting:` so the lines in the config file would look like: 
```
org.openhab.automation.jrubyscripting:config1=value
org.openhab.automation.jrubyscripting:config2=value
org.openhab.automation.jrubyscripting:config3=value
```

By adding this line, we can simplify the config file syntax to just 
```
config1=value
config2=value
config3=value
```

We'd much prefer the simpler syntax.

There are a few addons with this `configfile`: influxdb, openhabcloud, rrd4j, etc.

However, we defer to you for the final decision. Please advise.

There's also the question of naming. Currently we use the name `jruby.cfg` whereas the bundle's name is `jrubyscripting`. May we stick to `jruby.cfg` for the config file name or should we change it to `jrubyscripting.cfg` for uniformity's sake?